package.json

```diff
@@ -61,7 +62,9 @@
     "ldapjs": "1.0.0"
   },
   "devDependencies": {
+    "autoprefixer": "6.4.1",
     "chai": "3.5.0",
+    "cssnano": "3.7.5",
```

Comment:
I just played with cssnano a bit. It turns out our `style.css` is 30k, but minifying with `cssnano` makes it 23k.
Compared to our 289k `libs.min.js` and 55k lounge.templates.js`, I think this is a drop in the water. I honestly think we don't need this extra dependency + extra step and just keep the CSS as is until it becomes a problem.
What do you think?